ConsoleApp/Library/Customer.cs

```diff
@@ -0,0 +1,12 @@
+namespace ConsoleApp.Library;
+
+class Customer
+{
+    public string Name {get; init;}
+    public enum _sex
+    {
+        male,
```

Comment:
values pascal case as well
```suggestion
        Male,
```